src/protocol.rs

```diff
@@ -502,6 +527,12 @@ impl<'a> fmt::Display for Protocol<'a> {
             Ip6(addr) => write!(f, "/ip6/{}", addr),
             P2pWebRtcDirect => f.write_str("/p2p-webrtc-direct"),
             P2pWebRtcStar => f.write_str("/p2p-webrtc-star"),
+            WebRTC => f.write_str("/webrtc"),
+            Certhash(hash) => write!(
+                f,
+                "/certhash/u{}",
+                multibase::Base::Base64Url.encode(hash.to_bytes())
```

Comment:
```suggestion
                "/certhash/{}",
                multibase::encode(multibase::Base::Base64Url, hash.to_bytes())
```

How about the above? That way one does not  need to explicitly write out the `Base64Url` identifier (`u`).